### DISCUSSION TITLE
installing frepple 6.16.0 docker image

### DISCUSSION THREAD

**Discussion by snelstim** [frePPLe/frepple#389] [Q&A]
Hi, 

Can anyone guide me with installing frepple 6.16.0 with a docker image? I'm new to docker so I'm not sure what I'm doing and the information on the website is very confusing.
currently I'm stuck in following situation. I'm running "docker run ghcr.io/frepple/frepple-community:latest" I'm getting a message "waiting for the database to be ready" and then it stops.
not sure what to do from here.

Thanks in advance

**Comment by jdetaeye** [comment]
The frepple docker container needs a connection to a postgres database.
  
See https://frepple.org/docs/current/installation-guide/docker-container.html for a description of 2 methods to do this:
- install postgres on the docker host (or any other server)
- use a container to deploy postgres and use docker-compose to link the frepple and postgres containers

**Reply by snelstim** [reply]
thanks for the reply.
I tried to do docker-compose up and I got following error message:

ERROR: Named volume "config--community:/etc/frepple:rw" is used in service "frepple" but no declaration was found in the volumes section.

**Reply by jdetaeye** [reply]
That looks like a typo in your compose file.
Check out the spelling for this line;
     - config-community:/etc/frepple

The file in our documentation also has the typo with the -- double dash, and you have an additional :rw

**Reply by wangweity** [reply]
![image](https://user-images.githubusercontent.com/41356352/139363731-34bf9353-8059-4a78-8bb2-1c002feb4653.png)
the different between these two place is right for docker compose?

**Reply by jdetaeye** [reply]
Oops.  It's definitely not intended...
Thanks for reporting this!
